samples/tracing/sample.md

```diff
@@ -1,48 +1,48 @@
 ---
 title: Distributed tracing with Azure Monitor Application Insights
-summary: How to extend NServiceBus pipeline and send trace information to Azure Monitor Application Insights
+summary: How to extend the NServiceBus pipeline to send trace data to Azure Monitor Application Insights
 component: Core
-reviewed: 2020-09-04
+reviewed: 2020-09-10
 related:
  - monitoring/metrics
 ---
 
 ## Introduction
 
-.Net Core 3.1 introduces support for distributed tracing. Dedicated types in the [System.Diagnostics](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics?view=dotnet-plat-ext-3.1) namespace enable exposing and correlating execution information in a distributed system. This sample shows how to extend NServiceBus pipeline with custom behaviors that publish tracing information and how to push that data to [Azure Application Insight](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) for further processing and visualization. 
+.NET Core 3.1 introduced support for distributed tracing. Types in the [System.Diagnostics](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics?view=dotnet-plat-ext-3.1) namespace may be used to expose and correlate execution data from a distributed system. This sample shows how to extend the NServiceBus pipeline with custom behaviors that publish trace data and how to send that data to [Azure Monitor Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) for analysis and visualization.
 
-## Running the sample 
+## Running the sample
 
-Running the sample requires Application Insights resource. [Instrumentation Key](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-resources-app-insights-keys?view=azure-bot-service-4.0#instrumentation-key) has to be provided by setting `ApplicationInsightKey` environment variable.
+Running the sample requires an Application Insights resource. The [Instrumentation Key](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-resources-app-insights-keys?view=azure-bot-service-4.0#instrumentation-key) must be provided by setting the `ApplicationInsightsKey` environment variable.
 
-After starting, the sample requires pressing <kbr>Enter</kbr> to generate `InitialCommand` that when handled results in `FollowUpEvent` publication. Trace of each execution is pushed to Application Insights and available as [end-to-end transaction](https://docs.microsoft.com/en-us/azure/azure-monitor/app/transaction-diagnostics#transaction-diagnostics-experience) trace.
+After starting the sample, pressing <kbd>Enter</kbd> sends an `InitialCommand`. When that command is handled, a `FollowUpEvent` is published. Trace data for both actions is pushed to Application Insights and available as an [end-to-end transaction](https://docs.microsoft.com/en-us/azure/azure-monitor/app/transaction-diagnostics#transaction-diagnostics-experience) trace:
```

Comment:
> [end-to-end transaction](https://docs.microsoft.com/en-us/azure/azure-monitor/app/transaction-diagnostics#transaction-diagnostics-experience) trace

I don't see any mention of "end-to-end" in the linked page.

Reply:
Good catch. I've added suggestion. It used to be called end-to-end transactions - you can still see that in at the top of the screenshot that the link points to.